pkg/generator/marc.go

```diff
@@ -224,7 +224,7 @@ func marcToRecord(fmlRecord fml.Record, rules []*record.Rule, languageCodes map[
 		}
 		eightSixSix := getHoldings(fmlRecord, "866", []string{"b", "c", "h", "a", "z"})
 		if len(eightSixSix) > 0 {
-			r.Holdings[i].Summary += " " + eightSixSix[i].Summary
+			r.Holdings[i].Summary += " " + eightSixSix[0].Summary
```

Comment:
I think 866 is repeating. I'm okay with accepting this for now, but we may want to open a ticket to consider whether we need to handle multiple 866 fields in the future.

Reply:
I ran the entire Alma export on silent consumer with a Println for _any_ 866 field...we don't have a lot of them to begin with, and there was never more than one per record. I know theoretically we could get one in the future but I'm hoping we'll be switching to XML soon enough that it won't matter, and for that we'll need to redo and confirm all our mappings anyway.